usecases/backup/coordinator.go

```diff
@@ -665,3 +670,16 @@ type partialStatus struct {
 	*StatusResponse
 	err error
 }
+
+func CompressionTypeFromLevel(c CompressionLevel) (backup.CompressionType, error) {
+	switch c {
+	case GzipBestCompression, GzipBestSpeed, GzipDefaultCompression:
+		return backup.CompressionGZIP, nil
+	case ZstdBestCompression, ZstdBestSpeed, ZstdDefaultCompression:
+		return backup.CompressionZSTD, nil
+	case NoCompression:
+		return backup.CompressionNone, nil
+	default:
+		return "", fmt.Errorf("invalid compression level: %q", c)
```

Comment:
nit: I would not quote the compression level, a string (`%s`) IMO is better for search